### ISSUE TITLE
Clip the chart values instead of letting them go out of the chart bounds.

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#58]
![image](https://user-images.githubusercontent.com/679486/40331232-e292b982-5d0c-11e8-9837-027258eceefa.png)

**Comment by samreid** [comment]
Pushed a proposed fix, @arouinfar can you please review?

**Comment by arouinfar** [comment]
@samreid the clipping looks correct to me, but I think the bigger issue is that the wave itself should be better scaled vertically. At max amplitude. the peak of the wave should reach the top of the chart, like so:


![image](https://user-images.githubusercontent.com/8419308/43789782-1b474488-9a2e-11e8-90c6-11cc14a25b55.png)

**Comment by samreid** [comment]
I'm not sure that would be best, because the current behavior is like so:

![image](https://user-images.githubusercontent.com/679486/44592417-c0ab4300-a77d-11e8-88ab-1e5b461aed12.png)

If we cut the scaling back to say 60% or so, then the waves on the right side of the screen will be much smaller.  I'm not saying that the current scaling is perfect, but I think if we shrink it so much that the wave source always fits in the chart, then the rest of the waves or the right hand side will likely be difficult to read.

Thoughts @arouinfar?

**Comment by arouinfar** [comment]
@samreid agreed that scaling down by 60% would be pretty harsh.  For very large amplitudes, I could live with some of the curve getting clipped. I wouldn't want to see any clipping at the default amplitude, so we we may need to bump down the default by a tick mark.

Let's try:
 
* Scale down the curve a bit -- let's try 20%
* Increase the height of the graph so that the "Position" label is a few px above the bottom of the viewing window.

![image](https://user-images.githubusercontent.com/8419308/44690648-e2742680-aa18-11e8-87d3-a0aac1eab5d9.png)

**Comment by samreid** [comment]
I would like to proceed with scaling to 80%, but I recalled that we decided to match the scaling of the graph with the water side view, like so:

![image](https://user-images.githubusercontent.com/679486/44699967-e2d5e700-aa43-11e8-9bf9-9053e459d118.png)

So this means the water side view will also go to 80%.  If this is not what you intended, please let me know.  It shouldn't be a difficult change.

**Comment by samreid** [comment]
> Increase the height of the graph so that the "Position" label is a few px above the bottom of the viewing window.

I made this change, but had forgotten there was other code centering the chart in the bottom half of the wave area, so this inadvertently moved the top tab higher.  So now the default settings look like this:

![image](https://user-images.githubusercontent.com/679486/44700317-b622cf00-aa45-11e8-85db-9e48a4d84897.png)

and this:

![image](https://user-images.githubusercontent.com/679486/44700328-c175fa80-aa45-11e8-82bb-bd46f63083b2.png)

I could rewrite the part that centers the graph in the bottom half of the wave area, but I thought I'd run this past you first in case it looks good.  @arouinfar can you please take a look on phettest?

**Comment by arouinfar** [comment]
@samreid I think the scaling is looking definitely looking better. I would leave the graph centered in the bottom half, as it would look a bit odd otherwise.

One thing I’ve noticed is that the source point on the graph is very sharp, but otherwise is nicely sinusoidal. This is particularly noticeable in the 2nd screenshot in your most recent https://github.com/phetsims/wave-interference/issues/58#issuecomment-416444958. By comparison, Java looks much smoother at the source point. I realize that the sharp source point is not technically icorrect, since I imagine the wave originates from a single cell in the lattice. However, I wonder if Java had some extra smoothing applied to the curve. 
![image](https://user-images.githubusercontent.com/8419308/44919897-2e1e1d00-ad04-11e8-8590-bd9d2c2942b0.png)


The reason I bring this up is because it seems like the curve would be better contained in the graph if source point weren’t so sharp.

**Comment by samreid** [comment]
The Java version is using spatial averaging over a vertical stack of 3 cells:

```java
            double sum = getYORIG( index, yValue + 1 ) + getYORIG( index, yValue - 1 ) + getYORIG( index, yValue );
            return sum / 3;
```

Would you like to try that for HTML5?  Or maybe try vertical spatial averaging only near the source?  Please advise.

**Comment by arouinfar** [comment]
@samreid spatial averaging sounds promising. My gut says it'd only be necessary near the source, though I don't imagine it'd do any harm if it was in effect everywhere. I'll leave it up to you, in case there is some significant advantage/disadvantage to limiting the vertical spatial averaging to the region around the source.

**Comment by samreid** [comment]
On hold while we investigate #137
